### ISSUE TITLE
Add support for `content` field grabbing by rss plugin

### ISSUE THREAD

**Issue by chaosmaker** [Flexget/Flexget#656]
Hi there developers,

I have an issue that I cannot seem to resolve. I'm trying to extract links from a relatively detailed rss feed.

My current config looks like this:

```
tasks:
  Music Download:
    inputs:
      - rss:
          url: http://www.trancefix.nl/external.php?type=RSS2&forumids=210
          other_fields: ['content:encoded']  
          link: 'content:encoded'
    exec:
      - echo text="\"{{description}}\"" >> "/volume1/pyload/crawljob/{{title}}.crawljob"
      - echo downloadFolder="\"D:\\Downloads\\Trance\\\"" >> "/volume1/pyload/crawljob/{{title}}.crawljob"
```

However when I try and run this, flexget either doesn't recognise the field name or cannot read the rather long field "content:encoded" or it has a problem with the ":". I'm not sure but the output is the following:

```
2015-11-09 18:44 VERBOSE  task_queue                    There are 1 tasks to execute. Shutdown will commence when they have completed.
2015-11-09 18:44 WARNING  rss           Music Download  Skipped 2 RSS-entries without required information (title, link or enclosures)
2015-11-09 18:44 WARNING  inputs        Music Download  Input rss did not return anything
2015-11-09 18:44 WARNING  details       Music Download  Task didn't produce any entries. This is likely due to a mis-configured or non-functional input.
2015-11-09 18:44 VERBOSE  details       Music Download  Summary - Accepted: 0 (Rejected: 0 Undecided: 0 Failed: 0)
```

Any help would be appreciated!

**Comment by gazpachoking** [comment]
There are a couple things.
1. you don't want to set the `link` option to that field. You can always manipulate the url field later if need be. That's what's preventing your entries from even being created `Skipped 2 RSS-entries without required information (title, link or enclosures)`
2. Seems feedparser (the library rss plugin is using) does some special stuff with the content field. In your example, that field content can be accessed from python like `feedparseritem['content'][0]['value']`. The rss plugin would need to be modified somehow to support this before it could work.

**Comment by chaosmaker** [comment]
Hi thanks for your response. 
I initially did have it set up like this:
```
tasks:
  Music Download:
    inputs:
    rss:
      url: http://www.trancefix.nl/external.php?type=RSS2&forumids=210
      other_fields: ['content:encoded']
    manipulate:
      - description:
          from: 'content:encoded'
          extract: \b(([\w-]+://?|www[.])[^\s()<>]+(?:\([\w\d]+\)|([^[:punct:]\s]|/)))
    exec:
      - echo text="\"{{description}}\"" >> "/volume1/pyload/crawljob/{{title}}.crawljob"
      - echo downloadFolder="\"D:\\Downloads\\Trance\\\"" >> "/volume1/pyload/crawljob/{{title}}.crawljob"
```

but yes accessing the content:encoded field never worked. I wonder though why content:encoded is an issue but torrent:magnetURI isn't in terms of accessing it?

**Comment by gazpachoking** [comment]
> I wonder though why content:encoded is an issue but torrent:magnetURI isn't in terms of accessing it?

Because feedparser is doing special handling on that field to normalize it from different kinds of feeds I think.

**Comment by liiight** [comment]
Please reopen in [feathub](http://feathub.com/Flexget/Flexget)

**Comment by thawn** [comment]
I submitted a pull request that fixed the issue for me: #1880
